src/pkg/filters/filters.go

```diff
@@ -215,18 +365,67 @@ func prefixed(target, input string) bool {
 	return strings.HasPrefix(input, target)
 }
 
+// true if target is an _element complete_ prefix match
+// on the input.  Element complete means we do not
+// succeed on partial elementt matches (ex: "/foo" does
+// not match "/foobar").
+//
+// As a precondition, assumes the target value has been
+// passed through normPathElem().
+//
+// The input is assumed to be the complete path that may
+// have the target as a prefix.
+//
+// Basic principle: For any normalized, well formed target
+// and input (ex: values like "/a/b/c/"), the target is a
+// prefix if the regex `^target` matches the input.
+func pathPrefix(target, input string) bool {
+	re, err := regexp.Compile("^" + target)
+	if err != nil {
+		return false
+	}
+
+	return re.MatchString(normPathElem(input))
```

Comment:
since the regex pattern is fixed to the norm-ed CLI input with the exception of `^` can't this be simplified as `strings.HasPrefix(normPathElem(input), target)`?

Reply:
I suppose it does.